raiden/tests/unit/transfer/test_node.py

```diff
@@ -0,0 +1,37 @@
+from raiden.tests.utils.factories import EMPTY_MERKLE_ROOT, HOP1, HOP2, UNIT_SECRETHASH
+from raiden.transfer.events import ContractSendChannelBatchUnlock
+from raiden.transfer.node import is_transaction_effect_satisfied
+from raiden.transfer.state_change import ContractReceiveChannelBatchUnlock
+
+
+def test_is_transaction_effect_satisfied(
+        chain_state,
+        token_network_state,
+        token_network_id,
+        netting_channel_state,
+):
+    transaction = ContractSendChannelBatchUnlock(
+        token_network_identifier=token_network_id,
+        channel_identifier=netting_channel_state.identifier,
+        merkle_tree_leaves=EMPTY_MERKLE_ROOT,
+    )
+    state_change = ContractReceiveChannelBatchUnlock(
+        transaction_hash=UNIT_SECRETHASH,
+        token_network_identifier=token_network_id,
+        participant=HOP1,
+        partner=HOP2,
+        locksroot=EMPTY_MERKLE_ROOT,
+        unlocked_amount=0,
+        returned_tokens=0,
+    )
+    # try calling with the address being neither us nor our partner
```

Comment:
could you clarify this? e.g. `unlock for a channel which this node is not a participant must return False`